app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,75 @@
+class TweetsController < ApplicationController
+
+
+    get '/tweets/new' do
+      if logged_in?
+        erb :'/tweets/create_tweet'
+      else
+        redirect to "/login"
+      end
+    end
+
+    get '/tweets' do
+      if !logged_in?
+        redirect to "/login"
+      else
+        @user = current_user
+        @tweets = Tweet.all
+        erb :'/tweets/tweets'
+      end
+    end
+
+    post '/tweets' do
+      if logged_in? && !params[:content].empty?
+        @tweet = Tweet.create(content: params[:content], user_id: session[:user_id])
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.

Reply:
I didn't know you could chain those methods like that-- great tip! Thank you!